pages/sales.vue

```diff
@@ -8,7 +8,7 @@
   </section>
 </template>
 <script lang="ts">
-import { salesVisible } from '@/utils/config/permision.config'
+import { salesVisible } from '~/utils/config/permission.config'
```

Comment:
just keep it using `@` instead of `~`

Reply:
@preschian oops sorry my editor automatically renamed the files didn't notice it was changed to ~
Thank you for pointing that out